js/index.js

```diff
@@ -1 +1,36 @@
 // Your code goes here
+
+
+// click
```

Comment:
these comments are good for separation and readability, try to also add comments that explain what you're trying to do or want to do